bin.src/csc_command.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
```

Comment:
There is no longer a need to use `.py` for command line tooling. The user doesn't care that it's in python.

In general we prefer thin command-line implementations with the actual code for the command line distributed within the python package. This makes it easier to test and easier to extract any sphinx documentation from it.

See https://developer.lsst.io/stack/argparse-script-topic-type.html#the-autoprogram-directive

Reply:
This has been addressed